### PR TITLE
servo 1.0.0.0

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate <manifest>`, where `<manifest>` is the name of the manifest you're submitting?
- [x] Have you tested your manifest locally with `winget install -m <manifest>`?

-----

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/3882)

### CODE REVIEW

**Comment by oxygen-dioxide** [comment]
Sorry, I can't see your picture. What I see is "Uploading image.png…"

**Review by ItzLevvie — APPROVED** [review]

File: manifests/servo/servo/1.0.0.0.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Id: servo.servo
+Publisher: servo
+Name: servo
+Version: 1.0.0.0
```
Comment (Croydon) [inline manifests/servo/servo/1.0.0.0.yaml:4]: Looks like a wrong version number

**Pull request closed without merging**